src/ui/dialogs/settings.ts

```diff
@@ -37,15 +37,38 @@ class StartPage extends Page {
 class ClientSection extends Section {
    protected generateContentElement(): JQuery {
       let contentElement = new List();
+      contentElement.append(new ListItem(
+         Translation.t('After_changing_this_option_you_have_to_reload_the_page'),
+         undefined,
+         undefined,
+         undefined,
+         undefined
+      ));
 
       contentElement.append(new ListItem(
          Translation.t('Language'),
-         Translation.t('After_changing_this_option_you_have_to_reload_the_page'),
+         undefined,
          undefined,
          undefined,
          this.getLanguageSelectionElement()
       ));
 
+      contentElement.append(new ListItem(
+         Translation.t('trusted_domains'),
+         Translation.t('one_domain_per_line'),
+         undefined,
+         undefined,
+         this.getTrustedDomainsElement()
+      ));
+
+	  contentElement.append(new ListItem(
```

Comment:
Support for wildcards would be more flexibel. E.g. you could allow `*.data.foobar.com` and don't have to allow subdomains for every domain.

Reply:
wildcards added